spec/main_spec.rb

```diff
@@ -0,0 +1,143 @@
+require_relative '../bin/main.rb'
+
+describe Game do
+  player = Game.new('Ikraam', 'Kubilay')
+  describe '#initialize' do
+    it 'Reurns an ' do
+      expect { Game.new('Ikraam', 'Kubilay') }.not_to raise_error(ArgumentError)
+    end
+  end
+  describe '#names' do
+    it 'Reurns an array with two inputs and 5 times' do
+      expect(player.names).to eq(%w[Ikraam Kubilay] * 5)
+    end
+  end
+  describe '#board' do
+    it 'returns an array with 9 integers of 0' do
+      expect(player.board).to eq(Array.new(9, 0))
+    end
+  end
+  describe '#greeting' do
+    it 'returns the same as player.board passed to show_board_hint' do
+      expect(player.greeting).to eq(show_board_hint(player.board))
+    end
+  end
+end
+
+describe TicTacToe do
+  let(:x_wins) { [1, 1, 1, 0, 0, 0, 0, 0, 0] }
+  let(:y_wins) { [10, 10, 10, 0, 0, 0, 0, 0, 0] }
+  let(:draw) { [1, 1, 10, 10, 10, 1, 1, 10, 1] }
+  start_game = TicTacToe.new('Ikraam', 'Kubilay')
+  describe '#initialize' do
+    it 'Returns an ' do
+      expect { TicTacToe.new('Ikraam', 'Kubilay') }.not_to raise_error
+    end
+  end
```

Comment:
- [ ]  The description for the initialize it block 'Returns an' looks unfinished add a little more to it explaining what does it return.